### PR TITLE
Create openfort.md

### PR DESCRIPTION
**Pull request by joalavedra**

### CODE REVIEW

File: arbitrum-docs/for-devs/third-party-docs/Openfort/openfort.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+title: 'How to onboard users in your game and make a gasless transaction'
+description: 'Learn how to use Openfort to integrate authentication, private key management, and account abstraction into your game on Arbitrum'
+author: Joan Alavedra
+sme: joalavedra
+sidebar_label: 'Openfort'
+---
+
+:::info Community member contribution
+
+Shoutout to [@joalavedra](https://github.com/joalavedra) for contributing the following [third-party document](/for-devs/third-party-docs/contribute)!
```
Comment (anegg0) [inline arbitrum-docs/for-devs/third-party-docs/Openfort/openfort.md:11]:
```suggestion
The following document was contributed by  [@joalavedra](https://github.com/joalavedra). Give them a shoutout if you find it useful!
```

File: arbitrum-docs/for-devs/third-party-docs/Openfort/openfort.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+:::
+
+**[<ins>Openfort</ins>](https://www.openfort.xyz/)** is a headless wallet solution that helps developers integrate authentication, private key management, and account abstraction into their dApps. By abstracting away the complexities of blockchain interactions, Openfort allows you to build Web3 applications with a Web2-like user experience, without requiring end users to manage private keys or hold cryptocurrency.
```
Comment (anegg0) [inline arbitrum-docs/for-devs/third-party-docs/Openfort/openfort.md:15]:
```suggestion
**[<ins>Openfort</ins>](https://www.openfort.xyz/)** is a headless wallet solution that helps developers integrate authentication, private key management, and account abstraction into their dApps. By abstracting away the complexities of blockchain interactions, Openfort allows you to build Web3 applications with a Web2-like user experience without requiring end users to manage private keys or hold cryptocurrency.
```

File: arbitrum-docs/for-devs/third-party-docs/Openfort/openfort.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+
+Openfort enables you to:
+- Implement secure authentication and key management 
+- Create and manage smart contract wallets
+- Enable gasless and sponsored transactions
+
+Check out **[<ins>Openfort's Documentation</ins>](https://www.openfort.xyz/docs)** to get started.
+
+## Openfort Dashboard
```
Comment (anegg0) [inline arbitrum-docs/for-devs/third-party-docs/Openfort/openfort.md:24]:
```suggestion
## Openfort dashboard
```

File: arbitrum-docs/for-devs/third-party-docs/Openfort/openfort.md [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+
+The Openfort Dashboard allows you to:
```
Comment (anegg0) [inline arbitrum-docs/for-devs/third-party-docs/Openfort/openfort.md:26]:
```suggestion
The Openfort dashboard allows you to:
```

File: arbitrum-docs/for-devs/third-party-docs/Openfort/openfort.md [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+:::info Openfort Dashboard
+
+Please check out the [<ins>docs</ins>](https://www.openfort.xyz/docs/) to learn more about using the Openfort Dashboard. Click [<ins>here</ins>](https://dashboard.openfort.xyz) to access the Openfort Dashboard.
+
+:::
+
+
+## How to implement a gasless transaction on Arbitrum using Openfort
+
+With Openfort, you can sponsor transactions for your users, meaning that in-game wallets don't need to have native tokens to execute transactions, such as minting an NFT. This guide will walk you through the process of implementing a gasless transaction to mint an NFT on Arbitrum.
```
Comment (anegg0) [inline arbitrum-docs/for-devs/third-party-docs/Openfort/openfort.md:43]:
```suggestion
With Openfort, you can sponsor transactions for your users, meaning that in-game wallets don't need native tokens to execute transactions, such as minting an NFT. This guide will walk you through the process of implementing a gasless transaction to mint an NFT on Arbitrum.
```

File: arbitrum-docs/for-devs/third-party-docs/Openfort/openfort.md [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+
+### 1. Import the NFT Contract
```
Comment (anegg0) [inline arbitrum-docs/for-devs/third-party-docs/Openfort/openfort.md:45]:
```suggestion
### 1. Import the NFT contract
```

File: arbitrum-docs/for-devs/third-party-docs/Openfort/openfort.md [hunk 7]
```diff
@@ -0,0 +1,145 @@
...
+
+First, you need to import the smart contract you'll be interacting with. In this case, we'll use an NFT contract with a 'mint' function.
```
Comment (anegg0) [inline arbitrum-docs/for-devs/third-party-docs/Openfort/openfort.md:47]:
```suggestion
First, you need to import the smart contract you'll interact with. In this case, we'll use an NFT contract with a 'mint' function.
```

File: arbitrum-docs/for-devs/third-party-docs/Openfort/openfort.md [hunk 8]
```diff
@@ -0,0 +1,145 @@
...
+
+```bash
```
Comment (anegg0) [inline arbitrum-docs/for-devs/third-party-docs/Openfort/openfort.md:49]:
```suggestion
```shell
```

File: arbitrum-docs/for-devs/third-party-docs/Openfort/openfort.md [hunk 9]
```diff
@@ -0,0 +1,145 @@
...
+curl https://api.openfort.xyz/v1/contracts \
+  -u "$YOUR_SECRET_KEY:" \
+  -d 'name=NFT Contract' \
+  -d 'chainId=42161' \
+  -d 'address=YOUR_CONTRACT_ADDRESS'
+```
+
+Replace `YOUR_CONTRACT_ADDRESS` with the address of your NFT contract on Arbitrum.
+
+### 2. Set up the Gas Policy
```
Comment (anegg0) [inline arbitrum-docs/for-devs/third-party-docs/Openfort/openfort.md:59]:
```suggestion
### 2. Set up the gas policy
```

File: arbitrum-docs/for-devs/third-party-docs/Openfort/openfort.md [hunk 10]
```diff
@@ -0,0 +1,145 @@
...
+
+Create a new policy to sponsor gas fees for users:
+
+```bash
```
Comment (anegg0) [inline arbitrum-docs/for-devs/third-party-docs/Openfort/openfort.md:63]:
```suggestion
```shell
```

File: arbitrum-docs/for-devs/third-party-docs/Openfort/openfort.md [hunk 11]
```diff
@@ -0,0 +1,145 @@
...
+curl https://api.openfort.xyz/v1/policies \
+  -H "Authorization: Bearer $YOUR_SECRET_KEY" \
+  -d chainId=42161 \
+  -d name="Arbitrum NFT Sponsor" \
+  -d "strategy[sponsorSchema]=pay_for_user"
+```
+
+Then, add a policy rule for the NFT contract:
+
+```bash
```
Comment (anegg0) [inline arbitrum-docs/for-devs/third-party-docs/Openfort/openfort.md:73]:
```suggestion
```shell
```

File: arbitrum-docs/for-devs/third-party-docs/Openfort/openfort.md [hunk 12]
```diff
@@ -0,0 +1,145 @@
...
+curl https://api.openfort.xyz/v1/policies/:id/policy_rules \
+  -H "Authorization: Bearer $YOUR_SECRET_KEY" \
+  -d type="contract_functions" \
+  -d functionName="mint" \
+  -d contract="con_..."
+```
+
+Replace `:id` with the policy ID returned from the previous step, and `con_...` with your contract ID.
+
+### 3. Create a Gasless Transaction
```
Comment (anegg0) [inline arbitrum-docs/for-devs/third-party-docs/Openfort/openfort.md:83]:
```suggestion
### 3. Create a gasless transaction
```

File: arbitrum-docs/for-devs/third-party-docs/Openfort/openfort.md [hunk 13]
```diff
@@ -0,0 +1,145 @@
...
+  optimistic: true,
+  interactions: {
+    contract: 'con_....', // Your NFT contract ID
+    functionName: 'mint',
+    functionArgs: ['0x...'] // Address to receive the NFT
+  }
+});
+```
+
+### 4. Optional: Specify the Player
```
Comment (anegg0) [inline arbitrum-docs/for-devs/third-party-docs/Openfort/openfort.md:105]:
```suggestion
### 4. Optional: specify the player
```

File: arbitrum-docs/for-devs/third-party-docs/Openfort/openfort.md [hunk 14]
```diff
@@ -0,0 +1,145 @@
...
+    contract: 'con_....', // Your NFT contract ID
+    functionName: 'mint',
+    functionArgs: [playerId] // Minting to the player's address
+  }
+});
+```
+
+By following these steps, you've created a gasless transaction on Arbitrum using Openfort. The user can now mint an NFT without needing to hold ETH for gas fees. Remember to handle the response from the `transactionIntents.create` call in your application to provide feedback to the user about the minting process.
+
+:::info Detailed Tutorial
```
Comment (anegg0) [inline arbitrum-docs/for-devs/third-party-docs/Openfort/openfort.md:131]:
```suggestion
:::info Detailed tutorial
```

**Review by anegg0 — APPROVED** [review]
lgtm!
You just need to validate your CLA, and that article should be ready to merge :)

**Comment by joalavedra** [comment]
Thanks @anegg0 I did sign the CLA as well. Let me know if it's fine for you or I need to complete it again. Thanks

**Comment by hkalodner** [comment]
@cla-bot check

Sorry about the delay, but we're all set on the CLA, thanks!

**Comment by anegg0** [comment]
Merged! Thank you for contributing @joalavedra!

**Pull request merged**